react-nextjs/nextjs-host-react-remote/host/next.config.js

```diff
@@ -1,32 +1,26 @@
+const { NextFederationPlugin } = require('@module-federation/nextjs-mf');
+
 module.exports = {
-  webpack5: true,
   webpack(config, options) {
-    const { webpack, isServer } = options;
-    config.experiments = { topLevelAwait: true };
-
-    config.module.rules.push({
-      test: /_app.js/,
-      loader: '@module-federation/nextjs-mf/lib/federation-loader.js',
-    });
+    if (!options.isServer) {
+      config.plugins.push(
+          new NextFederationPlugin({
+            name: 'host',
+            remotes: {
+              remote: 'remote@http://localhost:8081/remoteEntry.js',
+            },
+            shared: {
```

Comment:
react does not need to be listed as shared here, its already done by default

Reply:
Haven't thought of that, fixed. Thanks!